pkg/minikube/service/service.go

```diff
@@ -275,7 +275,7 @@ func WaitAndMaybeOpenService(api libmachine.API, namespace string, service strin
 	chkSVC := func() error { return CheckService(namespace, service) }
 
 	if err := retry.Expo(chkSVC, time.Duration(interval)*time.Second, time.Duration(wait)*time.Second); err != nil {
-		return errors.Wrapf(err, "Could not find finalized endpoint being pointed to by %s", service)
+		return errors.Wrapf(err, "Service %s was not found in %s namespace , please try with 'minikube service %s -n Y'", service, namespace, service)
```

Comment:
If I understand properly, "Y" doesn't have a special meaning. What do you think about something like this:
```
Service %s was not found in the %q namespace. You may select another namespace by using:

'minikube service %s -n <namespace>'
```

Reply:
Make sense. PR has been updated. Thanks